spec/models/concerns/digital_object_concerns/validations_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe DigitalObjectConcerns::Validations, type: :model do
+  let(:item_invalid_uid) { FactoryBot.create(:item, uid: 'FFF') }
+  let(:item_uid_caps) { FactoryBot.create(:item, uid: '2F4e2917-26f5-4d8f-968c-a4015b10e50f') }
+  let(:item_valid) { FactoryBot.create(:item, uid: '2f4e2917-26f5-4d8f-968c-a4015b10e50f') }
+
+  context "when a uid is provided on digital object create" do
+    it "rejects an invalid uid" do
+      expect { item_invalid_uid }.to raise_error ActiveRecord::RecordInvalid, 'Validation failed: Uid is invalid'
+    end
+    it "rejects a valid uid with uppercase letters" do
+      expect { item_uid_caps }.to raise_error ActiveRecord::RecordInvalid, 'Validation failed: Uid is invalid'
+    end
+    it "accepts a valid uid" do
+      expect(item_valid).to be_valid
+    end
+  end
```

Comment:
Proposing a slightly different version of these tests:
```suggestion
  let(:valid_uid) { '2f4e2917-26f5-4d8f-968c-a4015b10e50f' }
  let(:invalid_uid) { 'FFF' }

  context "when a uid is provided on digital object create" do
    it "rejects an invalid uid" do
      expect { FactoryBot.create(:item, uid: invalid_uid) }.to raise_error ActiveRecord::RecordInvalid, 'Validation failed: Uid is invalid'
    end
    it "rejects a capitalized version of a valid uid" do
      expect { FactoryBot.create(:item, uid: valid_uid.upcase) }.to raise_error ActiveRecord::RecordInvalid, 'Validation failed: Uid is invalid'
    end
    it "accepts a valid uid" do
      FactoryBot.create(:item, uid: valid_uid)
    end
  end
```
Feel free to take what you think is useful in here (or not!).  Here's my thinking behind the proposal:
- This is personal preference, but when a creation operation is the thing that result in an error, I like to see the create call in the test rather than having to look at an earlier `let` to see what operation is raising an error.
- I thought that it might be illustrative to show the `upcase` call on the same `valid_uid` that passes in another test.